PMurphy1940/HandPickt
# Lifting plant state

# Description
The My Garden view is fully functional. User plants are displayed in a vertically scrolling view, sorted by closest to harvest.

## Type of change

```Plant State``` has been lifted to the new ```Page Views```  route to allow Plant Data sharing between My Garden and Archive views.


# Testing Instructions

 The user may ```Inspect``` or ```Remove``` a plant from the main list. Remove will give the option to ```Archive``` the plant along with a note, or just outright delete it from the database. Inspect will bring the user to the plants details view which will show an expanded card with full details on the plant. The user may edit the comments and/or the planting date only. Clicking anywhere on either field will active edit mode for that field only. Editable fields are indicated by the 'Pencil and Note' icons. Clicking the check after editing will turn off the edit field, but chages are retained. Days remaining to estimated harvest as well a progress pie chart are displayed as well. Clicking the pie will allow the user to force early harvest if the plant reaches it ahead of schedule.
If any fields are edited, a ```Save Changes``` button will display at the bottom of the view, just above the Nav Bar. The ```Back``` button will close the expanded view and discard any changes. ```Remove``` will do same as above.

# Checklist:
